eng/InternalTools.props

```diff
@@ -6,6 +6,10 @@
           https://devdiv.pkgs.visualstudio.com/_packaging/Engineering/nuget/v3/index.json;
           https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json;
         </RestoreSources>
+        <RestoreSources>
```

Comment:
Backport RestoreSource from #73532

Reply:
It doesn't look like there's any meaningful change - only whitespace removed. I'm fine with that, but wondering if that is intentional?